### ISSUE TITLE
Having Agent reacting to the presence of Park and Usage

### ISSUE THREAD

**Issue by agrignard** [agrignard/ReChamp#93]
After spending time in the exhibition there was a recurrent question: do agent react to the fact that we display or not park and usage? => the answer is no so far but I think it would make actually much more sense.

Not sure how to effectively implement that and that might be tricky as the population need to be updated every change but I think we could think of a structure where people switch to strool_in_park state only if the park are here and same for usage.

What do you think? does it make sense?

**Comment by tnguyenh** [comment]
I don't see why agents wouldn't react to parks when they are not displayed... parks are still there, they have not disappeared. To me the real problem is that displaying/hiding the parks has no real use, as it is a static layer, contrary to cars and other mobility modes.

**Comment by agrignard** [comment]
then we could illustrate the benefit of having park or not (no park no one hanging around and as soon as you trigger part the people start to stroll) this is more like a storytelling open question (again with the feedback that I had from the people in front of the table)

**Comment by tnguyenh** [comment]
I don't really see the point about that. What if we had removed parks ? Is someone really interested in that question ? Moreover this would be confusing. The bottom buttons are for display, the top one is for scenarii. That would be mixing scenarii and layers display.

In my opinion, the button for parks is quite useless (as is the usage button) appart from allowing to remove unnecessary informations for car and mobility layers.

**Comment by agrignard** [comment]
well a city without park means people don't stroll and just go from A to B as soon as you have park and usage it becomes more vibrant as peopel stop. Again it's a story telling question and a way to involve more the user instead of just showing or not the park you can really show the benefit or not of having it. 
There is no good or bad answer (and it might imply to redesign the physical interface also)

it doesn't need to be implemented right now in any case, just food for though

**Comment by tnguyenh** [comment]
Yes I had perfectly understood this, but still I don't see the point in adding a "remove park" scenario for the story telling... Anyway it is not difficult to implemant, just put a probability to go to a park to 0 when there are removed, and ask pedestrian to choose a new target if they are strolling in a park. In that case, the physical interface should be redesigned. If you want to be consistent, then you have to do the same for usage (but I don't think people have the same problem since the usage are still represented in the 3D model)...

My opinion since the very begining is that having buttons just to hide/display layer is confusing and does not bring much to the user experience, and it is still my line now. The fact that visitors raised this problem illustrate that it is actually confusing. But introducing a park/no park scenario would be just a patch that does not bring any useful information (keep in mind that it should be focused on the actual situation and the proposal from PCA, so why remove parks ?).

We sadly didn't have time to improve this part, but if you want to improve that, it is better to think about a better design of the whole user interface/interaction with the model. Isolating a layer by itself is not interesting. Adding more information (such as pollution or noise for cars) to each layer would already be a nice improvement.
